src/System.Management.Automation/CoreCLR/CorePsAssemblyLoadContext.cs

```diff
@@ -1,19 +1,14 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@iSazonov, your last commit had 2 failures in `PowerShell-CI-linux`
Can load a native assembly. Possitive test..Can load native dll
```powershell
Exception calling "LoadNative" with "0" argument(s): "Unable to load shared library '/tmp/tmpo23dLF.tmp/linux-x64/libSkiaSharp.so' or one of its dependencies. In order to help diagnose loading problems, consider setting the LD_DEBUG environment variable: /tmp/tmpo23dLF.tmp/linux-x64/libSkiaSharp.so: cannot open shared object file: No such file or directory"
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/Assembly.LoadNative.Tests.ps1: line 51
51:         [TestNativeClass]::LoadNative() | Should -Not -BeNullOrEmpty

```
Can load a native assembly.Can load native dll
```powershell
Expected an exception, with FullyQualifiedErrorId 'EntryPointNotFoundException' to be thrown, but the FullyQualifiedErrorId was 'DllNotFoundException'. from /home/vsts/work/1/s/test/powershell/engine/Basic/Assembly.LoadNative.Tests.ps1:119 char:11
    +         { [TestNativeClass2]::LoadNative() } | Should -Throw -ErrorId ?
    +           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/Assembly.LoadNative.Tests.ps1: line 119
119:         { [TestNativeClass2]::LoadNative() } | Should -Throw -ErrorId "EntryPointNotFoundException"
```